pages/pipelines/notifications.md.erb

```diff
@@ -149,6 +149,19 @@ notify:
 
 In the example above, the notification will be sent to the 'sre' and 'announcements' channels in the 'buildkite-community' workspace, the 'monitoring' channel in the 'buildkite-team' workspace, as well as to the 'general' channel of all configured workspaces.
 
+You can add a custom message to the notification:
+
+```yaml
+notify:
+  - slack:
+      - "buildkite-community#sre"
+      message: "SRE related information here..."
+  - slack:
+      - "buildkite-community#announcements"
+      message: "General announcement for the team here..."
+```
```

Comment:
I don't think these examples are valid yaml.

Does this work?

```yaml
notify:
   - slack: "buildkite#announcements"
     message: "New release"
   - slack:
       channels:
         - "buildkite#deploy"
         - "buildkite#infrastructure"
       message: "New release of buildkite deployed, ${BUILDKITE_COMMIT} is now in ${ENVIRONMENT}"
```

Reply:
😅 good catch - have updated